### ISSUE TITLE
Rnetica on Macbook pro M1 (silicon arm64) ?

### ISSUE THREAD

**Issue by Simon-h20** [ralmond/RNetica#18]
Hi @ralmond ,

Apologies in advance if this is not the appropriate place for this question / issue.

I downloaded the pkg`RNetica_0.8-4.tgz`and installed using Rstudio's pkg installer. When I load the Rnetica library in Rstudio I receive the error below suggesting an 'incompatible architecture'. 

Wondering if there is something wrong with my configuration or if it is not possible to run Rnetica on M1 Macbooks? 

Specs:
Macbook pro M1, MacOS Monterey 12.3.1
R 4.1.2 silicon arm64 build
I run Netica 6.09 32 bit version - using crossover

> library(RNetica)
Loading required package: CPTtools
Error: package or namespace load failed for ‘RNetica’ in dyn.load(file, DLLpath = DLLpath, ...):
 unable to load shared object '/Library/Frameworks/R.framework/Versions/4.1-arm64/Resources/library/RNetica/libs/RNetica.so':
  dlopen(/Library/Frameworks/R.framework/Versions/4.1-arm64/Resources/library/RNetica/libs/RNetica.so, 0x0006): tried: '/Library/Frameworks/R.framework/Versions/4.1-arm64/Resources/library/RNetica/libs/RNetica.so' (mach-o file, but is an incompatible architecture (have 'x86_64', need 'arm64e'))

Thank you!

**Comment by ralmond** [comment]
Thanks for letting me know.  My mac (and I think the ones on r-universe.dev) is x86_64, so I don't have that problem.
Unfortunately, I think it is an upstream issue, that is I need a version of libnetica.a or netica.dylib which is arm64e ready.  It was on my list to contact Brent@norsys about bringing the Mac and Windows shared libraries up to date, so I will pass this upstream.  Unfortunately, I don't how long that will take.
I will get back to you if I get word from Brent, as I probably will need somebody to test or arm64e for me.

**Comment by Simon-h20** [comment]
Hi @ralmond , okay thanks for letting me know 👍

**Comment by Simon-h20** [comment]
Hi @ralmond , wondering if you have had any updates on this?
Thanks,
Simon

**Comment by ralmond** [comment]
No, I've been meaning to contact Brent on a number of issues, but just 
haven't quite found the time.  (Mainly, the bug fixes in the linux 
version of the API need to be propagated to the MacOS and Win 
versions).  He should know about the issue, but I'm not sure where it is 
on his queue.

If you have a licensed Mac Version of the Netica API, I'd contact him 
directly.  Because as soon as I can get a arm64 version of the 
libnetica.so, I should be able to update RNetica to work with it.

     --Russell

**Comment by Simon-h20** [comment]
Okay, will do, thank you
Simon
